docker/grafana/dashboards/dashboard.json

```diff
@@ -2281,7 +2323,7 @@
                   },
                   {
                     "params": [
-                      "1s"
+                      "10s"
```

Comment:
Looks like you are missing the `/10` for this one

Reply:
The diff doesn't show this, but the dashboard already had the `/10` and was incorrect before.

Reply:
My mistake